helm-charts/charts/kube-starrocks/values.yaml

```diff
@@ -51,8 +51,21 @@ starrocksOperator:
 starrocksCluster:
   # the name of starrockscluster cluster, if not set, the chart name will be used.
   name: ""
-  # the namespace of starrockscluster cluster, if not set, the release namespace will be used.
-  namespace: ""
+  # By default, starrockscluster will be deployed in the same namespace as the
+  # operator, and normally you can use helm --namespace to specify the namespace.
+  # This is used to create a different namespace for starrockscluster cluster with specified annotations.
+  namespace:
```

Comment:
what if the user uses an old version of values.yaml which gives starrocksCluster.namespace != "", will it report error when upgrades to this new version?

Reply:
yes, this will be a break change. And user need to update its values.yaml file